packages/cubejs-questdb-driver/test/QuestDriver.test.ts

```diff
@@ -66,4 +66,61 @@ describe('QuestDriver', () => {
       );
     }
   });
+
+  test('schema', async () => {
+    const schema = await driver.tablesSchema();
+
+    expect(schema['']).toEqual({
```

Comment:
Where are the tables `query_test`, `telemetry` and `telemetry_config` created?

Reply:
`query_test` comes from the `test('query', ...)` test. `telemetry` and `telemetry_config` are built-in tables created for the purposes of telemetry events: https://questdb.io/docs/reference/configuration/#telemetry